charts/qdrant/templates/pdb.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: qdrant
+  labels:
+    {{- include "qdrant.labels" . | nindent 4 }}
+spec:
```

Comment:
What do you think about making `unhealthyPodEvictionPolicy` (https://kubernetes.io/docs/tasks/run-application/configure-pdb/#unhealthy-pod-eviction-policy) configurable? It's still in beta, but already active by default.

Reply:
done too but with an option (my cluster is still in 1.25 😓 )